### DISCUSSION TITLE
Use of pda

### DISCUSSION THREAD

**Discussion by xquery-go** [xquery-go/pallet-staking#2] [Q&A]
when I invoke the cpi, I have seed constraint error.
how can I resolve it?

**Comment by slycompiler** [comment]
Seed constraint error occurs when you use wrong pda accounts.
PDA: program derived accout are affected two factors: seed and program ID.
so when you invoke cpi, you have to use their program ID for PDA.

**Reply by xquery-go** [reply]
appreciate your help, now working well
